dallinger/recruiters.py

```diff
@@ -1,11 +1,15 @@
 """Recruiters manage the flow of participants to the experiment."""
 
 import os
+import traceback
 
 from boto.mturk.connection import MTurkConnection
 from psiturk.models import Participant
 from psiturk.psiturk_config import PsiturkConfig
 
+from dallinger import db
+session = db.session
```

Comment:
Does this assignment need to happen up here, vs. removing it and calling `db.session.commit()` on the line below?